features/tracker-allowlist.json

```diff
@@ -2516,7 +2516,13 @@
                             "royalcaribbean.com"
                         ],
                         "reason": "https://github.com/duckduckgo/privacy-configuration/issues/1255"
-                    }
+                    },
+                    {
+                        "rule": "tags.tiqcdn.com/utag/lbg/",
+                        "domains": [
+                            "<all>"
+                        ],
+                        "reason": "https://github.com/duckduckgo/privacy-configuration/issues/1334"
```

Comment:
```suggestion
                        "reason": "https://github.com/duckduckgo/privacy-configuration/issues/1334"
                    }
```

Reply:
Sorry, manually editing JSON is always error-prone...